### ISSUE TITLE
IntIntervalField can get invalid values.

### ISSUE THREAD

**Issue by tvuotila** [kvesteri/wtforms-components#47]
Following Intervals result in invalid or wrong field value when set to IntIntervalField:
 - Interval: `IntInterval(None)` hyphenized form: ` -` which is invalid value for  `IntInterval.from_string(' -')`
 - Interval `IntInterval([None, -3])` hyphenized form ` - -3` which is invalid value for `IntInterval.from_string(' - -3')`
 - Interval: `IntInterval([None, 3])` hyphenized form ` - 3` which gives `IntInterval('[-3, -3]')`

**Comment by tvuotila** [comment]
The problem seems to be that `IntIntervalField` thinks that `int_interval.hyphenized` and `IntInterval.from_string` are each other's inverse functions.

**Comment by kvesteri** [comment]
Good catch! Probably the best way to solve this would be to add from_hyphenized_string() classmethod to Interval class in ``intervals`` package.
